### ISSUE TITLE
Live View ! 

### ISSUE THREAD

**Issue by habibyuri** [jakiestfu/Bits#2]

**Comment by CredenceHamby** [comment]
Am i correct in saying you have to save the snip before seeing what it does?

**Comment by jakiestfu** [comment]
@EricHamby That is correct. I don't think it'd take that much work to pass in JS/HTML/CSS to a method which will build an iframe, though. You could even load a blank template in the iframe and traverse the DOM through the iframe since it's on the same domain.
